msticnb/nblib/azsent/host.py

```diff
@@ -104,72 +106,77 @@ def get_aznet_topology(
 
 @lru_cache()
 def verify_host_name(
-    qry_prov: QueryProvider, timespan: TimeSpan, host_name: str
-) -> Tuple[Optional[str], Optional[List[str]]]:
+    qry_prov: QueryProvider, start: datetime, end: datetime, host_name: str
+):
     """
     Verify unique hostname by checking Win and Linux logs.
 
     Parameters
     ----------
     qry_prov : QueryProvider
         Kql query provider
-    timespan : TimeSpan
-        Time span over which to query
+    start : datetime
+        Start of the time span over which to query
+    end : datetime
+        End of the time span over which to query
     host_name : str
         The full or partial hostname.
 
     Returns
     -------
-    Tuple[Optional[str], Optional[List[str]]]
```

Comment:
Note to self - I think I'll need to fix host_summary and win_event to handle the change in return type.